snowboots.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+Jeffrey Chen
+60 minutes
+7/10 test cases
+8/10 difficulty
+I used a greedy algorithm to solve it, which worked for some of the test cases but gave the wrong answer for other cases.
+*/

```

Comment:
That is a bad reflection. When you miss tests, you have error in your code or holes in your idea, that is nothing to do with what algorithm you use. It is algorithm problem when you have timeouts.